packages/x-charts/src/BarChart/useBarChartProps.ts

```diff
@@ -50,6 +50,8 @@ export const useBarChartProps = (props: BarChartProps) => {
     onHighlightChange,
     borderRadius,
     barLabel,
+    className,
+    ...rest
```

Comment:
The naming convention in the codebase?

```suggestion
    ...other
```

Reply:
Added changes to https://github.com/mui/mui-x/pull/13844